application/views/invoice_templates/pdf/InvoicePlane - overdue.php

```diff
@@ -35,23 +35,25 @@
             if ($invoice->client_city) {
                 echo htmlsc($invoice->client_city) . ' ';
             }
-            if ($invoice->client_state) {
-                echo htmlsc($invoice->client_state) . ' ';
-            }
             if ($invoice->client_zip) {
                 echo htmlsc($invoice->client_zip);
             }
             echo '</div>';
         }
+        if ($invoice->client_state) {
+            echo '<div>' . htmlsc($invoice->client_state) . ' ';
+        }
         if ($invoice->client_country) {
-            echo '<div>' . get_country_name(trans('cldr'), $invoice->client_country) . '</div>';
+            echo get_country_name(trans('cldr'), $invoice->client_country) . '</div>';
         }
 
```

Comment:
How is this correct? If either client_sate or country isn't defined, you will get either an non-closed div or you close 1 div too much?